test/models/IvdSubmissionSpec.scala

```diff
@@ -45,7 +44,7 @@ class IvdSubmissionSpec extends ModelSpecBase {
     defermentType = None,
     defermentAccountNumber = Some("1234567"),
     additionalDefermentAccountNumber = None,
-    supportingDocuments = Seq(
+    supportingDocuments = Vector(
```

Comment:
No need to change to Vector for here and line 57

Reply:
some tests fail because of this actually that's why the change to Vector